### PR TITLE
Refactor of code

### PR DESCRIPTION
**Pull request by Poostaq**
Changed node paths to export/assignment style 
Cleaned up signals - now point to one thing, but they are still not perfect 
Cleaned up unused private and public methods (underscore where necessary)
Changed some names so they more clearly describe what they do
Extracted a few additional methods, for code readability

Now attribute and ethnicity is set when ethnicity is switched (or when application is launched). Trait is set when trait button clicked, cleared up when ethnicity changed.

### CODE REVIEW

File: game/CharacterSheet/CharacterSheet.gd [hunk 1]
```diff
@@ -154,22 +161,25 @@ func _on_CloseButton_button_up():
 	self.mouse_filter = Control.MOUSE_FILTER_STOP
 
 
-func _on_Ethnicity_ethnicity_chosen(trait_button, ethnicity, attribute):
-	var trait_name
+func _on_Ethnicity_ethnicity_chosen(ethnicity):
+	self.character_stats_element.ethnicity = ethnicity["ethnicity_name"]
+
+
+func _format_trait_name(trait_button):
```
Comment (Malynna) [inline game/CharacterSheet/CharacterSheet.gd:168]: ethnicity trait

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 2]
```diff
@@ -1,7 +1,9 @@
 extends Control
 
-signal ethnicity_chosen(trait_element, current_ethnicity, bonus_attribute)
+signal ethnicity_chosen(current_ethnicity)
 signal attribute_chosen(bonus_attribute)
+signal trait_chosen(trait_element)
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:5]: ethnicity trait

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 3]
```diff
@@ -52,17 +59,15 @@ func load_ethnicity(_ethnicity):
...
+			var trait_button = _create_trait_button(alternate_trait_button_scene, trait)
 			var trait_button_trait_list = trait_button.get_node("MarginContainer/VBoxContainer/OptionButton")
-			fill_trait_button_trait_list(trait_button_trait_list)
+			_fill_trait_button_trait_list(trait_button_trait_list)
 			trait_button.secondary_trait = trait_button.option_button.get_item_text(0)
-			trait_button.get_node(".").set_button_group(trait_group)			
 		else:
-			var trait_button = create_trait_button(trait_button_scene, trait)
-			trait_button.get_node(".").set_button_group(trait_group)
+			_create_trait_button(trait_button_scene, trait)
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:67]: ethnicity trait

**Review by Malynna — APPROVED** [review]

File: game/CharacterSheet/CharacterSheet.gd (hunk 1 above)
Comment (Poostaq) [reply game/CharacterSheet/CharacterSheet.gd]: lets leave like that - its local anyway.

File: game/EthnicityPage/ChooseEthnicityStep.gd (hunk 2 above)
Comment (Poostaq) [reply game/EthnicityPage/ChooseEthnicityStep.gd:5]: changed method connected to signal which made more sense in result

File: game/EthnicityPage/ChooseEthnicityStep.gd (hunk 3 above)
Comment (Poostaq) [reply game/EthnicityPage/ChooseEthnicityStep.gd:67]: wouldnt help with clarity

**Pull request merged**